File: bug-tracker/src/main/java/rrpay/bug/service/BugService.java
package rrpay.bug.service;

import rrpay.bug.dto.BugDto;

import java.util.List;

public interface BugService {
    public List<BugDto> getBugs();

    public BugDto createBug(BugDto bug);

    public List<BugDto> getBugsByCode(String code);

    public BugDto getOneBugByCode(String code);

    public void deleteBug(String code);

    public void deleteAllBugs();

    public void updateBugDescription(String code, String description);
}

File: bug-tracker/src/main/java/rrpay/bug/kt/service/BugService.kt
package rrpay.bug.kt.service

interface BugService {

      getBugs():List<BugDTO>

      createBug(BugDTO bug):Bug

     getBugsByCode(String code): List<BugDTO>

     getOneBugByCode(String code): BugDTO

      deleteBug(String code):void

      deleteAllBugs():Unit

      updateBugDescription(String code, String description):String
}